apps/researcher/src/app/[locale]/research-guide/sortGuides.ts

```diff
@@ -0,0 +1,17 @@
+import {ResearchGuide} from '@colonial-collections/api';
+
+export function sortResearchGuide(topLevel: ResearchGuide): ResearchGuide {
```

Comment:
Curious: can we omit the return type (`ResearchGuide`), i.e. is TypeScript able to infer this?

Reply:
Yes, we can